float32 pan_force
int8 pan_direction
float32 tilt_force
int8 tilt_direction
